src/components/a-radio/_preview.html

```diff
@@ -1,2 +1,5 @@
-<axa-radio id="test" value="a" name="test-name"></axa-radio>
```

Comment:
This visible layout issues that need to be fixed before merging:

<img width="311" alt="bildschirmfoto 2018-02-16 um 09 18 09" src="https://user-images.githubusercontent.com/275235/36298870-924180a0-12fa-11e8-9d95-d23e64652498.png">